content/en/docs/concepts/scheduling/configure-scheduler.md

```diff
@@ -0,0 +1,223 @@
+---
+title: How to Configure Your Kubernetes Scheduler
+reviewers:
+- wgliang
+content_template: templates/concept
+weight: 30
+toc_hide: false
+---
+
+{{% capture overview %}}
+
+This is the third article on how to use and extend the Kubernetes scheduler series. The series of articles are listed below:
```

Comment:
Other docs references you can put in the final `What's Next`, not at the beginning.